go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -136,11 +134,12 @@ func (asf *ovnAddressSetFactory) DestroyAddressSetInBackingStore(name string) er
 	if err != nil {
 		return err
 	}
-	err = destroyAddressSet(getIPv4ASName(name))
+	ip4ASName, ip6ASName := MakeAddressSetName(name)
+	err = destroyAddressSet(ip4ASName)
 	if err != nil {
 		return err
 	}
-	err = destroyAddressSet(getIPv6ASName(name))
+	err = errors.Wrap(err, fmt.Sprintf("%v", destroyAddressSet(ip6ASName)))
```

Comment:
@JacobTanenbaum I applaud the attempt to simplify, but I think this would be clearer as it was before, not trying to roll the destroyAddressSet() into the errors.Wrap. Also, since the first error from the v4 address set destroy is returned, the errors.Wrap() here is useless.

Reply:
done